handlers/api.go

```diff
@@ -1952,6 +1952,7 @@ func getBalanceHistoryQueryParameters(q url.Values) (uint64, uint64, error) {
 		if err != nil || limit > defaultLimit || limit < 1 {
 			return 0, 0, fmt.Errorf("invalid limit parameter")
 		}
+		limit = limit - 1
```

Comment:
With this fix the function will return 0 as limit, which isn't true, as we want a limit of 1.

I would suggest change
`history, err := db.BigtableClient.GetValidatorBalanceHistory(queryIndices, latestEpoch-limit, latestEpoch)`
to
`history, err := db.BigtableClient.GetValidatorBalanceHistory(queryIndices, latestEpoch-limit+1, latestEpoch)`
instead, as it's the usage of limit which is wrong. 

Can be found here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d0a8c1c3a8b0819932784b2b3358d8721b195e2c/handlers/api.go#L1883

Reply:
@peterbitfly fyi

Reply:
> With this fix the function will return 0 as limit, which isn't true, as we want a limit of 1.
> 
> I would suggest change `history, err := db.BigtableClient.GetValidatorBalanceHistory(queryIndices, latestEpoch-limit, latestEpoch)` to `history, err := db.BigtableClient.GetValidatorBalanceHistory(queryIndices, latestEpoch-limit+1, latestEpoch)` instead, as it's the usage of limit which is wrong.
> 
> Can be found here:
> 
> https://github.com/gobitfly/eth2-beaconchain-explorer/blob/d0a8c1c3a8b0819932784b2b3358d8721b195e2c/handlers/api.go#L1883

You mean change to `latestEpoch-limit-1`?

Reply:
> You mean change to `latestEpoch-limit-1`?

I mean `latestEpoch-(limit-1)`, sorry

Reply:
Done

Reply:
Hm, which is the same. no I mean `latestEpoch-limit+1`

Reply:
`X - (L-1) = X - L + 1` just bad at math early in the morning :)